### PR TITLE
Export of PyNN models to NeuroML2

### PR DESCRIPTION
**Pull request by pgleeson**
Have tested this with simple PyNN file [here](https://github.com/OpenSourceBrain/PyNNShowcase/blob/master/PyNN/NeuroMLTest_PyNN0.8.py) (goes to [this](https://github.com/OpenSourceBrain/PyNNShowcase/blob/master/NeuroML2/PyNN_NeuroML2_Export.net.nml) and [this](https://github.com/OpenSourceBrain/PyNNShowcase/blob/master/NeuroML2/LEMS_Sim_PyNN_NeuroML2_Export.xml))

Also [this](https://github.com/OpenSourceBrain/Brunel2000/blob/master/PyNN/brunel08.py) [can be used](https://github.com/OpenSourceBrain/Brunel2000/blob/master/PyNN/brunel_to_neuroml.py) to generate [this](https://github.com/OpenSourceBrain/Brunel2000/blob/master/NeuroML2/BrunelFromPyNN.net.nml).

A slightly [modified version of the Potjans and Diesmann](https://github.com/pgleeson/PyNN/tree/test_potjans/examples/Potjans2014) can be exported to NML2 also, [here](https://github.com/OpenSourceBrain/PotjansDiesmann2014/tree/master/NeuroML2). This is just 0.1% of the full network and ~30MB in size... Will require save to [HDF5 format](https://github.com/NeuroML/NeuroML2/issues/93). XML version (144 cells 139,000 conns) can be viewed on OSB however:

selection_190

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,12 +14,5 @@ matrix:
...
-after_success:
-  - coveralls
-cache:
-  directories:
-    - $HOME/nest-2.10.0
-    - $HOME/nrn-7.4
-    - $HOME/build/nest-2.10.0
-    - $HOME/build/nrn-7.4
-    - $HOME/.cache/pip
+#after_success:
```
Comment (apdavison) [inline .travis.yml]: please could you revert the changes to the Travis config?

Comment (pgleeson) [inline .travis.yml]: Ok, I deleted those lines since I was having issues with tests passing due to the cache (now fixed) and interaction with coveralls. I'll see what I can do to get the coveralls check working with my fork

**Comment by apdavison** [comment]
Test failure is just a random statistical failure, unrelated to the PR

**Comment by pgleeson** [comment]
Thanks @apdavison. More changes to come (including of course some documentation), but this is the core of the export.
